Makefile

```diff
@@ -151,6 +152,9 @@ $(MOCKGEN): $(TOOLS_DIR)/go.mod # Build mockgen from tools folder.
 $(CONVERSION_GEN): $(TOOLS_DIR)/go.mod
 	cd $(TOOLS_DIR); go build -tags=tools -o $(BIN_DIR)/conversion-gen k8s.io/code-generator/cmd/conversion-gen
 
+$(KUSTOMIZE): $(TOOLS_DIR)/go.mod # Build kustomize from tools folder.
```

Comment:
If we're building kustomize here, do we still need to ensure and install [here](https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/master/scripts/ci-conformance.sh#L46)?

Reply:
Good catch, we do not.